infrasec/aws/aws-organizations.md

```diff
@@ -22,7 +22,7 @@ description of the patterns we've adopted.
 
 * Pick a standard prefix you can use for all the accounts in your
   organization; for our example we'll use `spacecats`.
-* The first account you should create is the "master" account for the
+* The first account you should create is the "main" account for the
```

Comment:
While I agree this shouldn't be "master", should this be something else like "primary" as it's referring to a totally different concept from branches?

Reply:
Honestly any word would work here. `main` is just as good as `primary`. From an infra perspective I don't think I actually use one or the other of those two examples.

Reply:
Kara is bringing up a good point here. This terminology comes directly from AWS and changing it to "main" is  nonsensical since it doesn't correlate with anything else about AWS Organizations nor is it in our control (compared to the default git branch) to change.

That said, what I think what we could do here is use something like "root", "organization root", or "org root". That concept _is_ part of AWS Organizations, so would line up with existing docs and our usage. It might make more sense to break that change out to a different PR so we could get some eyes on that piece from Chas and others who've been dealing a lot with AWS Organizations.

Reply:
I'll go with `org-root`